ansys/dpf/core/plotter.py

```diff
@@ -705,7 +707,7 @@ def plot_contour(
         kwargs.setdefault("nan_color", "grey")
 
         # Set the scalar bar title
-        kwargs.setdefault("stitle", name)
+        kwargs.setdefault("stitle", f"{name} ({unit})")
```

Comment:
Same here ?

Reply:
@anslpa same here, this defines the default stitle, then it is set in the following line